src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.ts

```diff
@@ -70,6 +81,7 @@ export class RgwBucketFormComponent extends CdForm implements OnInit, AfterViewC
     private rgwUserService: RgwUserService,
     private notificationService: NotificationService,
     private rgwEncryptionModal: RgwBucketEncryptionModel,
+    private textAreaJsonFormatter: TextAreaJsonFormatterService,
```

Comment:
Let's name it properly so we know its a service. and this applies to all other parts in the code.

```suggestion
    private textAreaJsonFormatterService: TextAreaJsonFormatterService,
```

Reply:
Got it